src/stest/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "stest"
+version = "0.0.0"
```

Comment:
When I run the new `stest` binary as `stest --version` it reports `0.0.0`. Can you have `clap` reference `env!("VERSION")` to get the version info when passed `--version`?

Reply:
You might need to override the `--version` command for this.

Reply:
Extremely pleased to discover that this just works:

```rust
#[command(author, version = env!("VERSION"), about, long_about)]
```

Clap really is awesome :)